playwright-e2e/settings/user-credentials.ts

```diff
@@ -28,6 +28,10 @@ export const judgeUser = {
   email: process.env.JUDGE_USERNAME || 'judiciary-only@mailnesia.com',
   password: defaultPwd,
 };
+export const judgeUserWithAdminRole = {
+    email: process.env.JUDGE_USER_WITH_ADMIN || 'judiciary@mailnesia.com',
+    password: defaultPwd,
+};
```

Comment:
I don't think we should be using this judge anymore, legacy judges will not have the correct roles moving forwards (due to interactions with AM + JRD), should be a CTSC user if admin actions like this event I think, or judge wales (or equiv) for judicial actions

Reply:
cool thanks, removed this user.